databricks_cli/configure/cli.py

```diff
@@ -31,6 +32,8 @@
 from databricks_cli.configure.config import profile_option, get_profile_from_context, debug_option
 
 PROMPT_HOST = 'Databricks Host (should begin with https://)'
+PROMPT_RESOURCE_ID = 'Resource/Workspace ID'
+PROMPT_AZ_TOKEN = 'Azure Token'
```

Comment:
nit: This field doesn't seem to be in use anymore.

Reply:
`PROMPT_AZ_TOKEN` is not being used anymore - removed it